docs/eigenda/operator-guides/protocol-SLA/protocol-SLA.md

```diff
@@ -50,15 +50,29 @@ Note: if the operator re-opts in the quorum at any point from `B` to `D`, the ab
 
 ## Accountability Measurements, Policies, and Actions
 
-### Liveness
+**Responsibilities**
 
-| Responsibility | SLI (measure) | SLA (policy) | Accountability (action) |
-| --- | --- | --- | --- |
-| Attesting | Signing rate: num-batches-signed / num-batches-responsible-to-sign | >=90% signing rate over rolling 48 hours (general) or >90% over rolling 24h (if operator has >4% stake) | Soft measure: Social pressure (if SLA is not violated) - Publish operator's attesting availability/performance and rank. <br/> Hard measure: Ejection (if SLA is violated) |
-| Serving | Serving availability: num-requests-success / num-total-requests | >=95% serving availability | Soft measure: Social pressure - Publish operator's serving availability/performance and rank. |
+Operators are responsible to perform both attestion and serving within the EigenDA protocol. We track performance of these responsibilities using the following service level indidactors (SLIs).
 
-### Safety
+| Responsibility | Daily SLI (measure) |
+| --- | --- |
+| Attesting | Signing rate: num-batches-signed / num-batches-responsible-to-sign |
+| Serving | Serving availability: num-requests-success / num-total-requests |
 
-| Responsibility | SLI (measure) | SLA (policy) | Accountability (action) |
-| --- | --- | --- | --- |
-| Attesting + Storing | Lazy signing rate: `num-batches-lazily-signed / num-batches-signed` | Lazy signing rate = `0` | Hard measure: slashing via Proof of Custody if SLA is violated (this is to be built). |
+Note that the SLI is evaluated over a 24 hour interval. 
+
+**SLA**
+
+When an operator is unavailable to perform its responsibilities, the damage to the network scales with the amount of take held by the operator. We ask operators to maintain high availability of both attesting and serving in accordance with the amount of stake delegated to the operator, as indicated by the following service level agreement (SLA) table. 
+
+| Share of Quorum Stake | Daily SLA (policy) | Nominal Maximum Daily Downtime |
```

Comment:
I thought these numbers are for attesting. For serving, it might be reasonable to have a higher availability requirement, as it's easier to meet in general.

Reply:
My impulse here is to leave this as simple as possible until we have data availability sampling. At the point, I think we'll need to be very nuanced with the retrieval SLA. What do you think?

Reply:
Ok, makes sense, this can be iterated later